docs/src/main/tut/index.md

```diff
@@ -105,7 +104,7 @@ import remotely._, codecs._
 object protocol {
   val definition = Protocol.empty
     .codec[Int]
-    .specify1[Int, Int]("factorial")
+    .specify1("factorial", Field.strict[Int]("n"), Type[Int])
```

Comment:
Ah, didn't notice that, thanks :)
Which form would you encourage for new users? That's what should go into the docs, I suppose :)